tests/runner.js

```diff
@@ -8,7 +8,7 @@ var glob = require('glob');
 var path = require('path');
 
 var root = 'tests/{acceptance,models}';
-var specFiles = glob.sync(root + '/**/*.spec.*');
+var specFiles = glob.sync(root + '/**/github-pages-deploy.spec.*');
```

Comment:
Can you remove this? I know why you added it, I do it myself while testing, but for merging it can't be there :D